doc/man/man3/crypto_argon2i.3monocypher

```diff
@@ -64,7 +64,8 @@ Must be
 .Fa nb_blocks
 × 1024 bytes big, and suitably aligned for 64-bit integers.
 .Fn malloc
-should yield a suitable chunk of memory.
+will yield a suitable chunk of memory on platforms that support 64-bit types
+(e.g. int64_t or IEEE double precision floats).
```

Comment:
Actually, it will work on *any* platform where Monocypher compiles at all. I believe this is written explicitly in the standard. For instance, on 32-bit platforms, `malloc()` may return a 4-byte aligned chunk of memory, but that's still enough to handle 64-bit integers: since they're represented by 2 32-bit words, we only need 32-bit alignment.

`will yield a suitable chunk of memory.` is enough.